crates/libcontainer/src/process/container_main_process.rs

```diff
@@ -229,6 +237,62 @@ fn setup_mapping(config: &UserNamespaceConfig, pid: Pid) -> Result<()> {
     Ok(())
 }
 
+/// setup_network_device sets up and initializes any defined network interface inside the container.
+fn setup_network_device(
+    linux: &Linux,
+    init_pid: Pid,
+    main_receiver: &mut channel::MainReceiver,
+    init_sender: &mut channel::InitSender,
+) -> Result<()> {
+    let mut addrs_map = HashMap::new();
+    // host network pods does not move network devices.
+    if let Some(namespaces) = linux.namespaces() {
+        if !namespaces
+            .iter()
+            .any(|ns| ns.typ() == LinuxNamespaceType::Network)
```

Comment:
You can use `contains`
```suggestion
        if !namespaces.contains(LinuxNamespaceType::Network) {
```